hierarqcal/utils.py

```diff
@@ -14,6 +14,7 @@
 import numpy as np
```

Comment:
The edges are sometimes off screen, there is xlim and ylims of the plot set on lines 96. I think you can just increase the epsilon value there to ensure the edges are always visible.

Reply:
Will update this.